packages/main/src/Tokenizer.ts

```diff
@@ -241,31 +241,33 @@ class Tokenizer extends UI5Element {
 		}
 	}
 
-	_delete(event) {
-		if (!event.detail) { // if there are no details, the event is triggered by a click
-			this._tokenClickDelete(event, event.target);
+	_delete(e: CustomEvent) {
```

Comment:
custom event here should be used as CustomEvent<Something> in order to point the delete event description from token.ts